automation/README.md

```diff
@@ -1,155 +1,131 @@
-# Automation with oVirt CI
-In order to keep playbooks in this repository operational,
-it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
-Everything located under `automation` directory is related to integration
-to [oVirt CI System][ovirt-ci-system-doc].
-In addition there is [`stdci.yaml`](../stdci.yaml) a configuration file for
+# Automation with oVirt CI 
+Any code changes in this repository** are verified using
 [oVirt CI System][ovirt-ci-system-doc].
 
-There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
-The KubeVirt Ansible repository uses the `check-patch` event at the moment,
-to verify incoming patches.
+All configurtion and script files required
+by [oVirt CI][ovirt-ci-system-doc] are located in the [automation](../automation)
+directory. In addition, there is the [`stdci.yaml`](../stdci.yaml) configuration file.
 
-Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
-triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
-which executes the [`automation/check-patch.sh`](./check-patch.sh) script.
+According to [oVirt CI][ovirt-ci-system-doc] specification all incoming patches
+are verified at the `check-patch` stage. Once a new PR is submitted to this repository,
+the [oVirt CI System][ovirt-ci-system-doc] triggers
+[Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity) which executes the [`automation/check-patch.sh`](./check-patch.sh) script.
+This script runs the Ansible playbook
+[`./playbooks/automation/check-patch.yml`](../playbooks/automation/check-patch.yml)
+which wraps the entire testing flow.
 
-This script will execute the Ansible playbook [`./playbooks/automation/check-patch.yml`](../playbooks/automation/check-patch.yml)
-which wraps entire testing flow.
+>**Note:** The [oVirt CI][ovirt-ci-system-doc] project is used to validate all
```

Comment:
AFAIK we are still far from there :-) ...

Reply:
@lukas-bednar Then I suggest to remove this phrase and original "oVirt CI is open to integration with .." completely, agree?

Reply:
Then I think rephrasing is wrong ...

That sentence was added there on the Eyal's request, and it wasn't meat to express that all kubevirt's components are validated with std-ci. It is supposed to say, that oVirt-CI is open to accommodate CI for any kubevirt's project. So one who maintains some kubevirt's component shouldn't hesitate to ask for help from oVirt-CI team.

Reply:
@lukas-bednar my rephrasing is wrong, true. But when I saw that phrase, I didn't understand it as what you wrote now. Your explanation now is more clear. The note about oVirt-CI for any KubeVirt related project and help from oVirt-CI Team is indeed a good one and I think placing it at the most beginning, right after the page Title is the best. And then it's good to give a link how to contact oVirt CI Team, their IRC or mailing list maybe. Agree?

Reply:
Sounds good!